activerecord/lib/active_record/migration/command_recorder.rb

```diff
@@ -108,11 +108,17 @@ def change_table(table_name, options = {}) # :nodoc:
         yield delegate.update_table_definition(table_name, self)
       end
 
+      def replay(migration)
+        commands.each do |cmd, args, block|
+          migration.send(cmd, *args, &block)
```

Comment:
Are we able to use `#public_send` here?

Reply:
Unfortunately, no. For example, this method https://github.com/rails/rails/blob/6556898884d636c59baae008e42783b8d3e16440/activerecord/lib/active_record/migration/command_recorder.rb#L116 is private.